client/components/tinymce/plugins/wpcom-view/views/simple-payments/index.jsx

```diff
@@ -19,15 +20,46 @@ import { getSelectedSiteId } from 'state/ui/selectors';
 import formatCurrency from 'lib/format-currency';
 import QuerySimplePayments from 'components/data/query-simple-payments';
 import QueryMedia from 'components/data/query-media';
+import QuerySitePlans from 'components/data/query-site-plans';
+import { hasFeature, getSitePlanSlug } from 'state/sites/plans/selectors';
+import { FEATURE_SIMPLE_PAYMENTS } from 'lib/plans/constants';
 
 class SimplePaymentsView extends Component {
 	render() {
-		const { translate, productId, product, siteId } = this.props;
+		const {
+			translate,
+			productId,
+			product,
+			siteId,
+			shouldQuerySitePlans,
+			planHasSimplePaymentsFeature,
+		} = this.props;
 
 		if ( ! product ) {
 			return <QuerySimplePayments siteId={ siteId } productId={ productId } />;
 		}
 
+		if ( shouldQuerySitePlans ) {
+			return <QuerySitePlans siteId={ siteId } />;
+		}
+
+		if ( ! shouldQuerySitePlans && ! planHasSimplePaymentsFeature ) {
+			return (
+				<div className="wpview-content wpview-type-simple-payments">
+					<div className="wpview-type-simple-payments__unsupported">
+						<div className="wpview-type-simple-payments__unsupported-icon">
+							<Gridicon icon="cross" />
+						</div>
+						<p
+							className="wpview-type-simple-payments__unsupported-message"
+							//eslint-disable-next-line react/no-danger
+							dangerouslySetInnerHTML={ { __html: "Your plan doesn't include Simple Payments." } }
```

Comment:
Why do we need to inject this dangerously?

i18n?

Reply:
Good catch!
Originally, this string was different based on the site type, and included a link (thus the danger).
When Editorial suggested this new simpler message, I just replaced it dumbly.